odoo/http.py

```diff
@@ -284,11 +284,25 @@ def get_default_session():
 class SessionExpiredException(Exception):
     pass
 
-def content_disposition(filename):
-    return "attachment; filename*=UTF-8''{}".format(
+
+def content_disposition(filename, disposition_type='attachment'):
+    """
+    Craft a ``Content-Disposition`` header, see :rfc:`6266`.
+
+    :param filename: The name of the file, should that file be saved on
+        disk by the browser.
+    :param disposition_type: Tell the browser what to do with the file,
+        either ``"attachment"`` to save the file on disk,
+        either ``"inline"`` to display the file.
+    """
+    if disposition_type not in ('attachment', 'inline'):
+        raise ValueError('Invalid disposition_type: %s' % (disposition_type))
```

Comment:
```suggestion
        e = "Invalid disposition_type: {disposition_type!r}"
        raise ValueError(e)
```
redacted text, hence double quotes
we prefer f-string to %s
`!r` to be closer to the ValueError and TypeError that python emits
since I'm at it, [EM102](https://docs.astral.sh/ruff/rules/f-string-in-exception/), not enforced yet but I know runbot loves this rule and wants to enforce it at some point

I feel very slimy, nitpicking for 4 different styling stuff that nobody (besides me) cares :grimacing: